API.md

```diff
@@ -443,7 +443,7 @@ Note, the response will have either uuids or 'GROUP' for the user ids, and a uui
 For descriptions of @msg_from, @msg_to and @ru see messages get
 [Get Message List](#get-message-list)
 
-## Get Conversations count  
+h2: Get Conversations count  
```

Comment:
I know this is super picky, but we use ## for a second level heading in every other repo, could we keep it consistent?

Reply:
sure